src/styles/index.ts

```diff
@@ -340,6 +340,9 @@ const styles = (theme: ThemeColors) =>
         appBG: {
             backgroundColor: theme.appBG,
         },
+        fontSizeLabel: {
+            fontSize: variables.fontSizeLabel,
+        },
```

Comment:
I think we can use the variables directly, instead of creating a style for it, e.g. [here](https://github.com/Expensify/App/blob/f8df4cf5a0068173a02201500c5672981e85fa15/src/pages/ReimbursementAccount/BankAccountStep.tsx#L181)